api-tests/api.test.js

```diff
@@ -21,6 +21,14 @@ expect.extend({
       : () => `expected ${received} to be a valid date string`
     return { pass, message }
   },
+  similarDate(recieved, expected) {
```

Comment:
Could you name this something closer to the built in jest matchers and the other ones we added?

Reply:
Sure